ScreenSleeper/Shared/Application.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Threading;
+
+namespace Shared
+{
+    public static class Application
+    {
+        private static readonly Mutex _mutex = new(true, name: "{04CF3278-CC61-4716-8B67-955C0FD7C469}");
```

Comment:
Might I suggest adding the third parameter (`createdNew`) to the constructor and using that instead of calling `WaitOne` for `IsRunning`? Of course, `IsRunning` will need to be renamed to `IsFirst` or something like that.

Reply:
Are you referring to the `CommandLineArguments` class constructor?

Reply:
Sorry, no, I mean the `Mutex` ctor on line 8. You can pass a `static bool` as an `out` parameter and the value will indicate if the `Mutex` was created (first instance) or not (second instance).

Reply:
Oh, ok. Good!